Was interesting.

Qsery93 opened the pull request:
- [DEMO LINK](https://QSERY93.github.io/layout_calendar/)

There is no report link for this exercise.

Qsery93 committed:
was interesting

qoa11a reviewed (changes requested):
![image](https://user-images.githubusercontent.com/102293862/201126365-ffaad1fd-cbbe-4bb3-b458-c504aaeb0d5f.png)
When I make 25 days in class than calendar disappears. Also add transition on hover to make it look better. :)

Qsery93 committed:
Update main.scss

Added transition 1s for the hover; 27 or less days wont make the calendar disapear.

marina-tilniak reviewed (changes requested):
fix checklist point 9 please

Qsery93 committed:
Update main.scss

marina-tilniak reviewed (changes requested):
перед тим як відправляти на повторне review потрібно перевірити чи все працює так як очікується, зараз анімація не плавна

Qsery93 commented:
Im sorry, i forgot to "npm run deploy" it, when i added changes here.

vlad-demchuk commented on src/styles/main.scss:
```diff
@@ -1,3 +1,72 @@
+    height: $cell_size;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
```
remove it to `*` selector

Qsery93 replied:
I dont understand what exactly i need to remove from there, because if i will remove selected lines, it will break all.

tetiana-ivanchenko replied:
box-sizing

vlad-demchuk commented on src/styles/main.scss:
```diff
@@ -1,3 +1,72 @@
+  @for $index from 1 through 31 {
+    &_cell:nth-child(#{$index})::before {
+      content: "#{$index}";
+      font-family: Arial, Helvetica, sans-serif;
```
remove it to `html` selector

vlad-demchuk commented on src/styles/main.scss:
```diff
@@ -1,3 +1,72 @@
+  &_cell:hover {
+    transform: translateY(-20px);
+    background-color: pink;
+    cursor: pointer;
```
checklist
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/102411130/201866179-d3d1e3c3-23af-4385-86e9-97592f61c197.png">

Qsery93 committed:
Update index.html

was added class html

Qsery93 committed:
Update main.scss

Qsery93 committed:
Update main.scss

Qsery93 committed:
Update main.scss

Qsery93 commented:
Я так відразу і зрозумів. Просто в мене після перенесення border-box все ламалось. Треба було ще видалити відступи і все запрацювало.

TarasHoliuk commented on src/styles/main.scss:
```diff
@@ -1,3 +1,77 @@
+    &--#{$day} &_cell:nth-child(-n + #{$day}) {
+      display: flex;
+    }
+  }
```
instead of set `display: none` by default set `display: flex` and set `display: none` in loop for the last days:
```
@for $index from 29 to 31 {
    &--#{$index} :nth-child(n + #{$index + 1}) {
      display: none;
    }
  }
```

Qsery93 replied:
Every time i learn something new. I didnt know that i can #{$index + 1}. I wrote #{$index} + 1 and it didnt work correct. So thats why i used -n and display: flex.

Qsery93 committed:
Update main.scss
